src/pgactive_apply.c

```diff
@@ -2122,7 +2126,11 @@ read_tuple_parts(StringInfo s, pgactiveRelation * rel, pgactiveTupleData * tup)
 	/* Consume remote data as long as there's a local column to put it in */
 	for (i = 0; i < Min(desc->natts, rnatts); i++)
 	{
+#if PG_VERSION_NUM >= 180000
+		FormData_pg_attribute *att = TupleDescAttr(desc, i);
```

Comment:
Okay, it's necessary due to 5983a4cffc31. That said I think that the check on the version is not needed and that we could make use of "TupleDescAttr" on all the version i.e making use of "&desc->attrs[i]" was not the most elegant way to get the attribute.  Standardizing on TupleDescAttr() usage would avoid the version check and make the code more readable.  This comment applies for all the changes related to TupleDescAttr() usage.